docs(mocktail): update `verify` example in `README.md` (#215)
felangel/mocktail: packages/mocktail/README.md

// Verify the interaction occurred.
verify(() => cat.sound()).called(1);

> // When mocktail verifies a method call, said call is then excluded from further verifications.

Reviewer: ```suggestion
// When mocktail verifies an invocation, it is then excluded from further verifications.
```

verifyNever(() => cat.sound());

// Interact with the mock instance again.
cat.sound();

> // The interaction count is now one again and not two.

Reviewer: ```suggestion
// The verification count is 1 since there was only 1 invocation since the last verification.
```
